templates/search.html

```diff
@@ -1,22 +1,25 @@
-{% if PLUGINS and 'tipue_search' in PLUGINS %}
-{% extends 'base.html' %}
-{% block title %}{{ _('Search') }} - {{ SITENAME|striptags }}{% endblock title %}
+{% if PLUGINS and 'tipue_search' in PLUGINS %} {% extends 'base.html' %} {%
```

Comment:
this seems odd, can you make {% foo %} in new lines? the same way it was before, it's easier to read

Reply:
Sorry this was autoformated. Will change it